bundles/org.openhab.binding.hydrawise/src/main/java/org/openhab/binding/hydrawise/internal/HydrawiseCloudHandler.java

```diff
@@ -127,9 +127,10 @@ protected void configure()
     protected void pollController() throws HydrawiseConnectionException, HydrawiseAuthenticationException {
         List<Controller> controllers = client.getCustomerDetails().controllers;
         Controller controller = getController(controllerId, controllers);
-        if (controller != null && !controller.online) {
-            throw new HydrawiseConnectionException("Controller is offline");
+        if (controller == null) {
+            throw new HydrawiseConnectionException("Controller is offline or missing");
```

Comment:
What about `controller.online`? Is it always assumed to be true then?

Reply:
It's no longer present in the API, i will remove it from the Controller class in the API package as well.